pkgs/top-level/all-packages.nix

```diff
@@ -9957,6 +9957,8 @@ with pkgs;
 
   jump = callPackage ../tools/system/jump { };
 
+  langgraph-cli = python3Packages.callPackage ../development/python-modules/langgraph-cli { };
```

Comment:
I'm wondering if this package shouldn't belong to `pkgs/by-name`.

Reply:
My past python applications submissions have gone her, per reviewiers

Reply:
What I would do if I were you, I would simply package it as a python module just like you did with other module in this PR, then use [`toPythonApplication`](https://github.com/NixOS/nixpkgs/blob/f32dd9ae988dd4c4cb1f2acdac84ae1273f1771b/doc/languages-frameworks/python.section.md?plain=1#L323) to create a simple binary in `pkgs/by-name`.

Reply:
Will do. Found https://discourse.nixos.org/t/ci-will-soon-enforce-pkgs-by-name-for-new-packages/38098 that documents the change. (Can you tell I'm fairly new at this? ;)